GetJobProfiles/Program.cs

```diff
@@ -195,6 +195,7 @@ static async Task Main(string[] args)
             //    await CopyRecipe(contentRecipesPath, "SharedContent");
             //    await CopyRecipe(contentRecipesPath, "ContentHelp");
                   await CopyRecipe(contentRecipesPath, "EmailTemplates");
+                  await CopyRecipe(contentRecipesPath, "ContactUsPages");
```

Comment:
If ContactUsPages is part of the pages release, it should be added to the `master_pages_release_39f603d6-746a-4324-b59a-31924e5d44fb.recipe.json` recipe instead of being part of the import util, and `ContactUsPages.recipe.json` placed in the `DFC.ServiceTaxonomy.Editor/Recipes` folder.